pkg/configuration/configuration.go

```diff
@@ -86,24 +89,40 @@ func New(configFilePath string) (*Registry, error) {
 
 func (c *Registry) setConfigDefaults() {
 	c.host.SetTypeByDefaultValue(true)
+	c.host.SetDefault(varDurationBeforeChangeRequestDeletion, "24h")
+
 	c.regService.SetTypeByDefaultValue(true)
 }
 
+// GetRegServiceImage returns the registration service image.
+func (c *Registry) GetDurationBeforeChangeRequestDeletion() time.Duration {
+	return c.host.GetDuration(varDurationBeforeChangeRequestDeletion)
+}
+
 // GetRegServiceImage returns the registration service image.
 func (c *Registry) GetRegServiceImage() string {
 	return c.regService.GetString(varImage)
 }
 
 // GetAllRegistrationServiceParameters returns the map with key-values pairs of parameters that have REGISTRATION_SERVICE prefix
 func (c *Registry) GetAllRegistrationServiceParameters() map[string]string {
+	return c.getAllParameters(RegServiceEnvPrefix)
```

Comment:
what about having a config specific to the registration service, backed by `viper` and set with a specific prefix, instead

Reply:
I don't understand what you mean by that?

Reply:
My question was: what about having 2 config objects, one for the host operator and one for the registration service, instead of collecting all env vars and filtering by prefix?

Reply:
> My question was: what about having 2 config objects, one for the host operator and one for the registration service

well, we already have something like that - we use two Viper configs with different prefixes:
https://github.com/codeready-toolchain/host-operator/blob/master/pkg/configuration/configuration.go#L54-L55

> instead of collecting all env vars and filtering by prefix?

we need to use different prefixes anyway. I guess that you just don't like the `getAllParameters` function. 
* For the registration-service it is necessary because we don't know which keys are going to be used there. So we need to get all env-vars and filter them by the prefix. After all, IIRC it was your idea :-) 
* for the host-operator, sure I don't really need this function because I know the keys so I can call directly the functions that return the specific values. It is just way easier to get all of the variables together with the keys.

Reply:
ok, let's address this in a separate PR, by refactoring the existing code.